scripts/dev.sh

```diff
@@ -1,30 +1,44 @@
 #!/bin/bash
 
+# Function to check if a command is available
+command_exists() {
+  command -v "$1" >/dev/null 2>&1
+}
+
+# Check if jq is installed
+if ! command_exists jq; then
+  echo "jq is not installed. Installing..."
+  sudo apt-get update
```

Comment:
This will only work on a small portion of OSs, so this code doesn't make sense until we have enough time to cover every case. I would propose to just leave the following link in the docs instead:

https://jqlang.github.io/jq/download/